static/app/debug/notifications/previews/slackPreview.tsx

```diff
@@ -1,9 +1,24 @@
+import {Link} from 'sentry/components/core/link';
 import {DebugNotificationsPreview} from 'sentry/debug/notifications/components/debugNotificationsPreview';
+import {
+  NotificationProviderKey,
+  type NotificationTemplateRegistration,
+} from 'sentry/debug/notifications/types';
+
+export function SlackPreview({
+  registration,
+}: {
+  registration: NotificationTemplateRegistration;
+}) {
+  const slackBlockKitBuilderBaseURL =
+    'https://app.slack.com/block-kit-builder/T023QCMBQG3#';
+
+  const blocks = JSON.stringify(registration.previews[NotificationProviderKey.SLACK]);
+  const blockKitPreviewLink = `${slackBlockKitBuilderBaseURL}${blocks}`;
```

Comment:
Is this preview functionality something we should consider baking into our providers or renderers instead of a standalone thing?